frontend/src/store/slice/VisualizeItem/VisualizeItemUtils.ts

```diff
@@ -45,6 +45,13 @@ export function isCsvItem(item: VisualaizeItemType): item is CsvItem {
   )
 }
 
+export function isMatlabItem(item: VisualaizeItemType): item is CsvItem {
+  return (
+    item.itemType === VISUALIZE_ITEM_TYPE_SET.DISPLAY_DATA &&
+    item.dataType === DATA_TYPE_SET.CSV
+  )
+}
+
```

Comment:
Does this appropriate?
And `isMatlabItem` may not be used in other codes.

Reply:
@ReiHashimoto i removed it, pls check it again